packages/atomic/src/components/insight/atomic-insight-generated-answer/atomic-insight-generated-answer.tsx

```diff
@@ -134,30 +144,21 @@ export class AtomicInsightGeneratedAnswer
   @AriaLiveRegion('generated-answer')
   protected ariaMessage!: string;
 
-  private generatedAnswerCommon!: GeneratedAnswerCommon;
   private fullAnswerHeight?: number;
+  private modalRef?: HTMLAtomicGeneratedAnswerFeedbackModalElement;
+  private setCopied = () => {
+    this.copied = false;
+  };
+  private setCopyError = () => {
+    this.copyError = false;
+  };
+  private _data: any = null;
 
   public initialize() {
-    this.generatedAnswerCommon = new GeneratedAnswerCommon({
-      host: this.host,
-      withToggle: this.withToggle,
-      collapsible: this.collapsible,
-      disableCitationAnchoring: this.disableCitationAnchoring,
-      getGeneratedAnswer: () => this.generatedAnswer,
-      getGeneratedAnswerState: () => this.generatedAnswerState,
-      getSearchStatusState: () => this.searchStatusState,
-      getBindings: () => this.bindings,
-      getCopied: () => this.copied,
-      setCopied: this.setCopied,
-      getCopyError: () => this.copyError,
-      setCopyError: this.setCopyError,
-      setAriaMessage: this.setAriaMessage,
-      buildInteractiveCitation: (props) =>
-        buildInsightInteractiveCitation(this.bindings.engine, props),
-    });
+    this._data = readGeneratedAnswerStoredData(!!this.withToggle);
```

Comment:
This makes more sense to be written like this, there's no point in reading the visible state in the storage if the `withToggle` isn't `true`.


```suggestion
    this._isVisible = !!this.withToggle && readGeneratedAnswerVisibilityState();
```